test/auth_cluster/test_raft_service_levels.py

```diff
@@ -59,13 +59,13 @@ async def test_service_levels_snapshot(manager: ManagerClient):
 @pytest.mark.asyncio
 @skip_mode('release', 'error injections are not supported in release mode')
```

Comment:
> We prefer a flag over an error injection because we will use it
> a lot in CI jobs' configurations, some tests, manual testing etc.
> It's much more convenient.

Another positive side effect of this is that this flag works in release mode. Please go over the tests you modified and see which ones should be re-enabled in release mode. Look at the `@skip_mode` annotations but also look at `skip_in_release` in `suite.yaml`.

Reply:
Good catch, I did it.